File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Prompt the user to type in the height he/she whats for the pyramid
    int h;
    do
    {
        h = get_int("Height: ");
    }
    while (h < 1 || h > 8); // The input must between 1 and 8
    // Print the pyramid
    int s = 7; // Define a variable for number of '#'
    int l = 0; // Define a variable for how many times the loop for one row runs
    while (l < h)
    {
        int i = 0; // The number of space before each line
        while (i < (h - l - 1))
        {
            printf(" ");
            i++;
        }
        int j = 0; // The number of '#'
        while (j < (8 - s))
        {
            printf("#");
            j++;
        }
        printf("  "); // The number of spaces is a constant
        int k = 0; // The opposite ones :)
        while (k < (8 - s))
        {
            printf("#");
            k++;
        }
        printf("\n");
        s--;
        l++;
    }
}

File: sentimental-mario-more/mario.py
# TODO
from cs50 import get_int

while True:
    h = get_int("Height: ")
    if h >= 1 and h <= 8:
        break
s = 7
l = 0

while l < h:

    i = 0
    while i < (h - l - 1):
        print(" ", end="")
        i += 1
    j = 0
    while j < (8 - s):
        print("#", end="")
        j += 1
    print("  ", end="")
    k = 0
    while k < (8 - s):
        print("#", end="")
        k += 1
    print()
    s -= 1
    l += 1